src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -267,6 +290,16 @@ private static IEnumerable<T> ReadFromProtobufStream<T>(Stream stream, MessagePa
             return new Result<T>(httpResponse, parseStringResult(stringResponse));
         }
 
+        private void ThrowsIfError(JObject parsedJson)
```

Comment:
We should probably came up with a better solution but the kind of return depends on the API and the SQ version so I went for the easy path.

Reply:
How difficult would be it be to add a unit test, at least for the API that was causing the failure?

Reply:
I will add more tests!